app/knights/heroes.py

```diff
@@ -0,0 +1,36 @@
+class Hero:
```

Comment:
It would be better to use `Knight` class name so that it would be more relevant to the application context